### PR TITLE
Week 1 bootcamp using Jetpack Compose + Material3

### PR DESCRIPTION
**Pull request by WollfieGitHub**
This adds the code generated by Android Studio using the Material3 + JetpackCompose Empty Activity preset.

The only modified code is in src/ and androidTest/, according to the Week 1 Bootcamp

### CODE REVIEW

**Review by Lynoxe — CHANGES_REQUESTED** [review]
I would suggest to add test tags directly to the interesting components rather than the column containing them. It would be more robust if we change the UI organisation (for example add a second element in the column between the `TextField` and the `Button`. Furthermore it would make the tests more self-explanatory (currently we don't know, by reading only the tests, that `column.onChildAt(0)` is the `TextField` for example). Having two separate test tags such as "UsernameField" and "StartButton" for example would be clearer.

**Review by Lynoxe — COMMENTED** [review]
I like the `object DeclaredIntents`, I didn't think about that myself.

**Review by Lynoxe — COMMENTED** [review]
The TextField isn't visible before we click on it, maybe add some background color or some indicator in some way would be nice.
However as this is only a sample scene that we'll remove it doesn't really matter for now.

**Review by Lynoxe — CHANGES_REQUESTED** [review]
Add the .idea folder to the .gitignore (I think it isn't important, just annoying)

File: app/src/main/java/com/github/leuludyha/ibdb/GreetingActivity.kt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+import com.github.leuludyha.ibdb.ui.theme.IBDBTheme
+
+class GreetingActivity : ComponentActivity() {
+
+    object DeclaredIntents {
+        const val username: String = "Username"
+    }
```
Comment (Lynoxe) [inline app/src/main/java/com/github/leuludyha/ibdb/GreetingActivity.kt:24]: I like this way to handle the intents extras, I didn't think about it!

**Review by Lynoxe — APPROVED** [review]
LGTM

**Review by HanaFarid — APPROVED** [review]
LGTM!

**Pull request merged**